[2022-08-30 00:30:22,736] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.start scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2022-08-30 00:30:22,754] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.start scheduled__2022-08-29T00:00:00+00:00 [queued]>
[2022-08-30 00:30:22,755] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-30 00:30:22,755] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-30 00:30:22,756] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-30 00:30:22,791] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): start> on 2022-08-29 00:00:00+00:00
[2022-08-30 00:30:22,800] {standard_task_runner.py:52} INFO - Started process 25292 to run task
[2022-08-30 00:30:22,805] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'start', 'scheduled__2022-08-29T00:00:00+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpgoeakjyq', '--error-file', '/tmp/tmp4716iyon']
[2022-08-30 00:30:22,807] {standard_task_runner.py:80} INFO - Job 278: Subtask start
[2022-08-30 00:30:22,954] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.start scheduled__2022-08-29T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-30 00:30:23,202] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=start
AIRFLOW_CTX_EXECUTION_DATE=2022-08-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-29T00:00:00+00:00
[2022-08-30 00:30:23,206] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-30 00:30:23,208] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'date']
[2022-08-30 00:30:23,267] {subprocess.py:85} INFO - Output:
[2022-08-30 00:30:23,286] {subprocess.py:92} INFO - Tue Aug 30 00:30:23 UTC 2022
[2022-08-30 00:30:23,294] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-30 00:30:23,415] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=start, execution_date=20220829T000000, start_date=20220830T003022, end_date=20220830T003023
[2022-08-30 00:30:23,500] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-30 00:30:23,594] {local_task_job.py:273} INFO - 4 downstream tasks scheduled from follow-on schedule check
